tempus_fugit_minecraft/player.py

```diff
@@ -149,7 +152,7 @@ def adjust_sight(self, dx, dy):
 
     def current_speed(self):
         return self.FLYING_SPEED if self.flying else self.walking_speed
-    
+
     def toggle_flight(self):
```

Comment:
Probably best to link the issue number and leave an embedded comment on the function.